rate-providers/registry.json

```diff
@@ -2315,6 +2343,28 @@
           "implementationReviewed": "0xe2D2E90122cb203CF1565a37ef90a256843A825A"
         }
       ]
+    },
+    "0xa65ffBa1CD05414df3fD24Bf5dc319B47450Fbf4": {
+      "asset": "0x1CE7D9942ff78c328A4181b9F3826fEE6D845A97",
+      "name": "YieldFi yUSD Rate Provider",
+      "summary": "unsafe",
```

Comment:
Let's wait for them to upgrade to multisigs, then we can merge this pr. Blocked until then I suggest.

Reply:
On a side note, should we start using `usable`/`unusable` in the registry as well?
If yes, maybe we should have a separate PR to make that a standard through all existing reviews and registry entries.